src/services/outliningElementsCollector.ts

```diff
@@ -200,6 +200,7 @@ namespace ts.OutliningElementsCollector {
             case SyntaxKind.EnumDeclaration:
             case SyntaxKind.CaseBlock:
             case SyntaxKind.TypeLiteral:
+            case SyntaxKind.ObjectBindingPattern:

```

Comment:
Add ArrayBindingPattern as well?

Reply:
@sheetalkamat I added `ArrayBindingPattern` in this PR https://github.com/microsoft/TypeScript/pull/38191